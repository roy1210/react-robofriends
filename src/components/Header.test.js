import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';
// import '../setupTests';

it('renders Header without crashing', () => {
  const components = shallow(<Header />);
  expect(components.debug()).toMatchSnapshot();
});
