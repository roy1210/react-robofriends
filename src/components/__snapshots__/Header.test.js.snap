// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header without crashing 1`] = `
"<div>
  <h1 className=\\"f2\\">
    RoboFriends
  </h1>
</div>"
`;
